org.eclipse.xtext.xtext.ui.examples/projects/fowlerdsl/org.eclipse.xtext.example.fowlerdsl/src/org/eclipse/xtext/example/fowlerdsl/GenerateStatemachine.mwe2

```diff
@@ -32,6 +32,7 @@ Workflow {
 				createEclipseMetaData = true
 			}
 			code = {
+				preferXtendStubs = false
 				encoding = "UTF-8"
 				lineDelimiter = "\n"
 				fileHeader = "/*\n * generated by Xtext\n */"
```

Comment:
No need for the fileheader to change here?

Reply:
this was inconsistent an i want to do this in a separate pr